### DISCUSSION TITLE
SC Lean Coffee 2024 calendar and virtual whiteboard is now live!

### DISCUSSION THREAD

**Discussion by mmoatts** [brianpetro/obsidian-smart-connections#721] [Announcements]
Hi everyone!

The calendar for upcoming Lean Coffee events is now live [here](https://lu.ma/calendar/manage/cal-ZJtdnzAdURyouM7). 

Additionally, we have a virtual whiteboard for these discussions [here](https://miro.com/app/board/uXjVKtX5Pu4=/?moveToWidget=3458764596746128023&cot=10), which is also where we post recording information (typically found in a green box at top right of each meeting canvas). 

Hope to see you soon!

- Matthew

**Comment by franciscoabenza** [comment]


**Reply by mmoatts** [reply]
Hi Francisco - The links seem to be working for me. Are they not taking you to the full year calendar of events for Lean Coffee and the Miro board respectively? If not, please share more details so I can help troubleshoot. Thanks!

**Comment by brianpetro** [comment]
😊🌴
